### ISSUE TITLE
TODO list

### ISSUE THREAD

**Issue by valitas** [BartKieviet/Bookkeeper#61]
When you log in after a while, and maybe also at building ticks while you're online, Bookkeeper pops up an overlay box on your screen telling you the most urgent things for you to do.  This box lists buildings low or out of upkeep.

There should be a way to configure which buildings appear in the box, and of course a way to disable this completely. In fact, we need to be very careful that popping up an unsolicited box on your screen, maybe while you're moving around, doesn't get you killed.

**Comment by valitas** [comment]
The convo that started this:

> Sweetness - Today at 13:28
> Is there anyway we can set up custom email alerts or anything?
> like, if 10% of the buildings I track in a system go to projecteds tock left of 1 tick I get an email
does that make sense what I am asking for?
> Vicki - Today at 13:30
> it does but... i don't think we can email you :/ bookie is a serverless utility, the only computer involved is your own. so your own computer would have to connect to some email server, to send you an email >.<
> what we could do is pop up a box in your pardus screen, telling you these things so when you next log in you know what's urgent..
> like, you haven't logged in in a while, so next time you do you get a big bookkeeper popup telling you "oi, you need to see to this"
> Sweetness - Today at 13:32
> Yeah, something like that would be nice, and have it be customized so I can have it only pop up if buildings I deem important are low on ticks.
> so like, my main deal right now is making sure EF never miss a tick because we run really lean on those
> it would be nice if when I logged in I got a pop up box that told me if my projections indicate any of the EFs i track are low.
> or something
> Vicki - Today at 13:34
> ok. telling bookie which buildings are "important" may be part of a different issue.. but yeah ok i'll try to summarise >.<
> Sweetness - Today at 13:36
> couldn't it just be a filter?
> on the alerts
> or w/e

The "telling Bookie which buildings are important" may be dependent on #38 (if we go for a per-building setting). Even if we don't and go for a filter-like setting, like Sw suggests above, it would be really nice to have #59, which does depend on #38.

**Comment by valitas** [comment]
And you know, this may be as simple as popping up the same overview box, prefiltered for ticks: `"Cor Caroli" 1`

Though I would like if it were a different-looking box, so the user knows this wasn't a misclick of theirs. Maybe a centered overlay with a title, "Urgent" or whatever, and including a prefiltered compact overlay. Bonus points for including the pic of Luca there 😆

**Comment by valitas** [comment]
Thinking further of this, I think the best way to do it would be to post a notification, like we do when we scan the sector index, telling you simply that "10 buildings are running low of upkeep". We can do this even if you're not playing Pardus, so long as your browser is running (see [`chrome.alarms`](https://developer.chrome.com/extensions/alarms)). And _then_, if you click on the notification, it should open the prefiltered overview.
